netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -439,7 +442,7 @@ public boolean visit(Http2Stream stream) throws Http2Exception {
   @Override
   public void handleProtocolNegotiationCompleted(
       Attributes attributes, InternalChannelz.Security securityInfo) {
-    this.attributes = attributes;
+    this.attributes = this.attributes.toBuilder().setAll(attributes).build();
```

Comment:
I don't understand this change.

Reply:
This merges the new attributes from the protocol negotiator into `this.attributes`.
`this.attributes` includes the EAG attrs. `attributes` is from the protocol negotiator and doesn't include EAG attrs.